src/workerd/api/actor.h

```diff
@@ -151,13 +158,20 @@ class DurableObjectNamespace: public jsg::Object {
     JSG_METHOD(idFromName);
     JSG_METHOD(idFromString);
     JSG_METHOD(get);
+    JSG_METHOD(getExisting);
```

Comment:
Should this be wrapped in a compat flag check? (I believe that would also hide it from the type definitions.)

Reply:
If calling the method produces a useful user facing error when it is not supported I'm not sure there's much value in hiding it with a compat flag. It's arguably more useful to try calling it and get a specific "this is a thing but it's not supported" error right now as opposed to a "getExisting is not a function" error.

Reply:
I don't agree. There is currently debate over whether this is functionality we want to provide at all, and the current proposal it to provide it just to one internal customer on an experimental basis. I don't think experimental APIs which we haven't committed to should show up for people who don't have the experimental flag enabled.

Reply:
> There is currently debate over whether this is functionality we want to provide at all

Oh, in that case, definitely compat flag :-) I missed that bit of context.

Reply:
Honestly, even if we were relative sure we wanted it, I think I'd still be for the compat flag until it's for general consumption. I've seen too many things that were a good idea at the time that didn't end up making it all end users.

Reply:
> ... I've seen too many things that were a good idea at the time...

/me looks at most of JavaScript .... mm hmm. yep.